src/components/ContentLine/ContentLine.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import ContentLine from './ContentLine';
+
+it('renders without crashing', () => {
+  const selectActiveWord = () => { };
+  const focusCustomInput = () => { };
+  const div = window.document.createElement('div');
+  render(
+    <ContentLine
+      selectActiveWord={selectActiveWord}
+      focusCustomInput={focusCustomInput}
+    >
+      children
+    </ContentLine>, div,
```

Comment:
When the arguments to `render()` are split over multiple lines, we should separate them by a new line. For example,

```react
  render(
    <ContentLine
      selectActiveWord={selectActiveWord}
      focusCustomInput={focusCustomInput}
    >
      children
    </ContentLine>,
    div,
  )
```

Could you change it here, `ContentWord.test.js`, `InputXml.test.js`, and `WordPanel.test.js`?